app/models/user.rb

```diff
@@ -3,8 +3,9 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
-  
+         CONST = 'hi   ' 
+         CONST2='s'
```

Comment:
Inconsistent indentation detected.<br>Freeze mutable objects assigned to constants.<br>Surrounding space missing for operator `=`.